.gitignore

```diff
@@ -0,0 +1 @@
+venv/
```

Comment:
What are we using `venv` for? Don't really mind it being included, but I'm wondering if it would be more appropriate in the excludes rather than the common ignore?